pack-plugin.php

```diff
@@ -30,7 +40,7 @@
 
 if ($zip->open($zipFile, ZipArchive::CREATE) !== true) {
     echo 'Can\'t open zip file.' . PHP_EOL;
-    exit(3);
+    exit(4);
```

Comment:
Have we released this yet? Compat break!

Reply:
released today, actually changed all the codes to 1 instead